tools/container/start-fwe.sh

```diff
@@ -10,6 +10,8 @@ mkdir -p `dirname ${CONFIG_FILE}`
 /usr/bin/configure-fwe.sh \
     --input-config-file /usr/share/aws-iot-fleetwise/static-config.json \
     --output-config-file ${CONFIG_FILE} \
+    --certificate "`echo "${CERTIFICATE}"`" \
```

Comment:
This change isn't required as the environment variables are already passed to `configure-fwe.sh`

Reply:
removed